docs/source/guide/prompts_draft.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Draft a prompt
+short: Draft a prompt
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 231
+meta_title: Draft a prompt
+meta_description: Create and evaluate an LLM prompt
+section: Prompts
+date: 2024-06-12 14:09:09
+---
+
+With your [model created](prompts_model), you can begin drafting prompts. 
+
+## Draft a prompt and generate predictions
+
+
+1. Select your base model. For a description of all OpenAI models, see [OpenAI's models overview](https://platform.openai.com/docs/models/models-overview).
+2. In the **Prompt** field, enter your prompt. Keep in mind the following:
+    * You must include the text class. (In the demo below, this is the `review` class.) Click the text class name to insert it into the prompt. 
+    * Although not strictly required, you should provide definitions for each class. 
+3. Click **Save**. 
+4. Click **Evaluate**. 
+
+!!! note
+    When you click **Evaluate**, you will create predictions for each ground truth task. When you return to the project, you will see this reflected in the project. You can see how many predictions a task has using the **Predictions** column in the Data Manager. 
```

Comment:
This will mostly be helpful for clarifying which records will be evaluated when you choose other baselines than "Ground Truth", but I think saying something like "...you will create predictions for each task included in the baseline you've selected" (then maybe linking to the section with info about the two different workflows, depending on the baseline the user has selected)